deploy/.env.example

```diff
@@ -26,6 +26,9 @@ FOREIGN_REQUIRED_BLOCK_CONFIRMATIONS=8
 FOREIGN_GAS_PRICE=10
 #for bridge erc to erc mode
 ERC20_TOKEN_ADDRESS=
+HOME_BRIDGEBLE_TOKEN_NAME=
```

Comment:
- [ ] Move these three lines to that part of the file which is related to `Home` network.
- [ ] Update README.md to reflect the changes (https://github.com/poanetwork/poa-bridge-contracts/blob/refactor_v1/deploy/README.md)

Reply:
I moved into this section under comment `#for bridge erc to erc mode`

Reply:
will do